### ISSUE TITLE
Potential issue with Get Response on Case4Short?

### ISSUE THREAD

**Issue by mornevdw** [danm-de/pcsc-sharp#28]
Hi,

I have a problem where I send a select application EMV command which is a CASE4SHORT. However the 6126 sw1w2 causes a Get Response but during this the le of the get respons is set to 0, not 0x26. This causes that to fail and so a 6C26 response is received.

I stepped through and changed the first line in ` private ResponseApdu IssueGetResponseCommand(..)` from:

`var le = (commandApdu.Le < lastResponseApdu.SW2)
                ? commandApdu.Le
                : lastResponseApdu.SW2; `

to
 
`var le = (commandApdu.Le > 0 && ( commandApdu.Le < lastResponseApdu.SW2 ))
                ? commandApdu.Le
                : lastResponseApdu.SW2;`

Basically, if le is 0, set it to SW2, otherwise choose the smallest value.

I am not sure if this is 100% correct and if it will break something else, but it solved my problem.

**Comment by danm-de** [comment]
Oh, same issue as described here https://github.com/danm-de/pcsc-sharp/pull/9/commits/7aae91e79b084eba23d71bc826b9b43f9410190b.

**Comment by danm-de** [comment]
New version 3.5.1. Thanks!

**Comment by mornevdw** [comment]
Wow! That was fast!
Thank you!
